OloEngine/src/OloEngine/Networking/Replication/EntitySnapshot.cpp

```
@@ -0,0 +1,162 @@
+        // Each transform is 9 floats = 36 bytes.
+        static constexpr u32 kTransformBytes = 9 * sizeof(f32);
+
+        std::unordered_map<u64, const u8*> baselineMap;
+        if (!baseline.empty())
+        {
+            FMemoryReader baseReader(baseline);
+            baseReader.ArIsNetArchive = true;
+
+            while (baseReader.Tell() < baseReader.TotalSize() && !baseReader.IsError())
+            {
+                u64 uuid = 0;
```

Clang (14.0.6)

[warning] variable 'uuid' is not initialized

(cppcoreguidelines-init-variables)